data.sql

```diff
@@ -55,3 +55,38 @@ UPDATE animals SET owner_id = (CASE
                                 WHEN name IN ('Charmander', 'Squirtle', 'Blossom', 'Angemon', 'Boarmon') THEN 4
                                 ELSE 5
                               END);
+
+
```

Comment:
- [ ] It would be nice if you make your PR Title such as `Vet clinic database: add 'join table' for visits` instead of `Database Update: Vets, Specializations, and Visits` to make your PR Title descriptive 👍
- [ ] It would be nice if you made your branch name such as `feature/join_tables` instead of `vet_clinic_database_update_v2` to make your pull request more professional and descriptive 😊